apps/website/src/pages/index.astro

```diff
@@ -83,10 +83,10 @@ const currentPage = new URL(Astro.request.url).pathname;
       </div>
     </div>
 
-    <div class='centered-content'>
+    <div class='centered-content move-on-visible' data-move-on-visible-direction='up'>
```

Comment:
instead of animating in multiple steps, this might be better if the whole section animates together. will look better on taller displays, and less distracting for anyone.

Reply:
I did it this way since I thought it looks better. I don't feel it is distracting. On taller displays, since items display one at a time, it might actually make each item stand out. Seeing all at once makes it look like too much content at once.

[Video16.webm](https://user-images.githubusercontent.com/45748283/214939266-b164d1ab-0194-415f-b373-36eaa0ca5b17.webm)

But I can change it if you feel it is distracting.

Reply:
well, of course it is not distracting if you slowly scroll a few pixels at a time, but it will be when you scroll a large amount all at once.

maybe you can add some kind of debouncing, like 50ms, and then decide how much to animate.

Reply:
I'm not sure how to apply debouncing to an animation. Or maybe you meant add debouncing to the `onScroll` listener? (Now I'm using `IntersectionObserver`)

For now, I just made the sections animate as one unit. So I removed the animation from the children of the "Why iTwinUI" and "Our Library" sections.

Reply:
i meant debouncing where you set the data attributes yeah. but animating the sections as one unit might be enough.

there is also a new [`scrollend`](https://developer.chrome.com/blog/scrollend-a-new-javascript-event/) event, available through a polyfill. it could be useful here